docs/guides/implementation-guides/how-to-implement-dependency-injection.md

```diff
@@ -85,7 +85,7 @@ class Program
 ```
 
 ## Step 5: Modify App.axaml.cs
-We are using the `ServiceManager` class to retrieve the right view model.
+Modiying the `App` class to use `ServiceManager` to retrieve instances of the data context with all their dependencies resolved via DI.
```

Comment:
This feels less like a tutorial and more like a command? You could instead phrase this as "Next, the `App.xaml.cs` class should be modified to use the DI container. This will allow the previously registered view model to be resolved via the dependency injection container. The fully realised view model can then be set to the data context of the main view. "